core/controlplane/config/templates/cloud-config-worker

```diff
@@ -146,9 +146,13 @@ coreos:
         --cluster_dns={{.DNSServiceIP}} \
         --cluster_domain=cluster.local \
         --cloud-provider=aws \
-        --kubeconfig=/etc/kubernetes/worker-kubeconfig.yaml \
+        {{- if .Experimental.ClusterTLSBootstrap.Enabled }}
+        --experimental-bootstrap-kubeconfig=/etc/kubernetes/worker-bootstrap-kubeconfig.yaml \
+        {{- else }}
         --tls-cert-file=/etc/kubernetes/ssl/worker.pem \
-        --tls-private-key-file=/etc/kubernetes/ssl/worker-key.pem
+        --tls-private-key-file=/etc/kubernetes/ssl/worker-key.pem \
```

Comment:
I guess the `experimental-bootstrap-kubeconfig` setting result in kubelet to write issued worker key and cert to `/var/run/kubernetes`, which is the well-known location for the cert and the key according to https://kubernetes.io/docs/admin/kubelet/

If that's true, I wonder if we could improve code to assume all the worker key and the cert are located under the well-known location, regardless of we enable the tls bootstrap feature or not. Doing so would require `/opt/bin/decrypt-tls-assets` to write decrypted key and cert to `/var/run/kubernetes/<the well-known name for key or cert, if any>` but result in various if-else statements be simplified; For example, this code block can be just:

```
{{- if .Experimental.ClusterTLSBootstrap.Enabled }}
--experimental-bootstrap-kubeconfig=/etc/kubernetes/worker-bootstrap-kubeconfig.yaml \
{{- end}}
```

i.e. explicit `tls-cert-file` and `tls-private-file` can be omitted when the tls bootstrap is disabled?